client/verifier.go

```diff
@@ -68,14 +72,14 @@ func (c *logVerifier) VerifyRoot(trusted, newRoot *trillian.SignedLogRoot,
 // the currently trusted root. The inclusion proof must be requested for Root().TreeSize.
 func (c *logVerifier) VerifyInclusionAtIndex(trusted *trillian.SignedLogRoot, data []byte, leafIndex int64, proof [][]byte) error {
 	leaf := c.buildLeaf(data)
-	return c.v.VerifyInclusionProof(leafIndex, trusted.TreeSize,
-		proof, trusted.RootHash, leaf.MerkleLeafHash)
+	return c.v.VerifyInclusionProof(leafIndex, trusted.GetTreeSize(),
```

Comment:
Ditto here and at VerifyInclusionByHash. I'd rather error out or panic, silently proceeding makes the `trusted = nil` misstep harder to detect.

Please add a test for VerifyInclusion* methods too, if you decide to touch them.